media-libs/libmediaart/libmediaart-1.9.0-r1.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+VALA_USE_DEPEND="vapigen"
+VALA_MIN_API_VERSION="0.16"
+
+inherit autotools eutils flag-o-matic gnome2 vala virtualx
```

Comment:
Looks like `eutils` is not used anymore since the ebuild uses EAPI 6 so you can drop it.